### PR TITLE
Create new note UI

### LINKED ISSUE #3: Create new note UI

### PR DESCRIPTION
**Pull request by vijay2136**
fixes #3 
- Created new note UI
- Created edit note UI

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Pane/CreateNote.jsx [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
 
-import { Pane, Typography } from "neetoui";
+import { Pane, Typography, Toastr } from "neetoui";
 
 import formInitialValues from "constants/formInitialValues";
 
 import Form from "./Form";
 
-export default function NewNotePane({ fetchNotes, showPane, setShowPane }) {
+const NewNotePane = ({ showPane, setShowPane }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/CreateNote.jsx:9]: This is wrong! Component file name doesn't match with Component name.

File: app/javascript/src/components/Dashboard/Notes/Pane/EditNote.jsx [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
 import Form from "./Form";
 
-export default function EditNotePane({
-  fetchNotes,
-  showPane,
-  setShowPane,
-  note,
-}) {
+const EditNotePane = ({ showPane, setShowPane }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/EditNote.jsx:9]: Why have you suffixed with keyword `Pane` when this is already within Pane folder?

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 3]
```diff
@@ -2,26 +2,17 @@ import React, { useState } from "react";
...
-import { Input, Textarea } from "neetoui/formik";
+import { Input, Textarea, Select } from "neetoui/formik";
 
-import notesApi from "apis/notes";
 import formValidationSchemas from "constants/formValidationSchemas";
 
-export default function NoteForm({ onClose, refetch, note, isEdit }) {
+import { ASSIGNED_CONTACT_OPTIONS, TAGS_OPTIONS } from "../constants";
+
+const NoteForm = ({ isEdit = false, onClose, note, handleNote }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:11]: What's your reasoning for naming `NoteForm` over just `Form`? I would like to understand why you are prefixing/suffixing keywords from time to time when naming components.

**Review by yedhink — COMMENTED** [review]
@vijay2136 

Few comments added.

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx (hunk 3 above)
Comment (vijay2136) [reply app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:11]: This file and above mentioned files were already present in wheel and somehow I missed to check the naming conventions. Will correct all the file names in next PR.

**Pull request merged**